platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/TransactionHandler.java

```diff
@@ -56,7 +56,7 @@ public void handleRound(final ConsensusRound round, final MerkleRoot state) {
             final Instant timeOfHandle = Instant.now();
             final long startTime = System.nanoTime();
 
-            state.getSwirldState().handleConsensusRound(round, state.getPlatformState());
+            state.getSwirldState().handleConsensusRound(round, state.getReadablePlatformState());
```

Comment:
Perhaps I'm missing something, but wouldn't handling a consensus round require a _writable_ state?

Reply:
Surprisingly enough, it's nod needed for consensus itself. However, you intuition is correct in case of tools. I'll update it.